### ISSUE TITLE
xmi encoding is not utf8

### ISSUE THREAD

**Issue by tpaviot** [buildingSMART/IFC_UML#1]
Current encoding is windows-1252:
```<?xml version="1.0" encoding="windows-1252"?>```

Could it be moved to utf8 ?

**Comment by aothms** [comment]
Good point, fully agree. I'm not the author of the XMI, I'll try to forward it to the appropriate people. cc @berlotti
